starter-code/pizza.js

```diff
@@ -1 +1,81 @@
 // Write your Pizza Builder JavaScript in this file.
+function main() {
+    // Iteration 1
+    addRemovePep();
+    addRemoveMushorooms();
+    addRemoveGreenPeppers();
+    // Iteration 2
+    glutenFreeCrust();
+    whiteSauce();
+    calculateTotalPrice();
+}
+
+function listIngredients(ingredient) {
+    $('aside>ul>li:contains('+ingredient+')').toggle();
+}
+
+function calculateTotalPrice() {
+    var ingredients = $('aside>ul>li');
+    var totalPrice = 10;
+    $(ingredients).each(function(index, li){
+        if($(li).is(":visible")) {
+            totalPrice += parseInt($(li).text().substring(1,2));
+        }
+    });
+    $('aside>strong').text("$" + totalPrice);
+}
+
+function addRemovePep() {
```

Comment:
As you said, you can make one general function and avoid repeating the code again and again.